File: wondes_journey_code_csharp/Character/Character.cs
public class Character
{
    private string name;
    private int life;
    private int damage;
    private bool hasAnEffect;
    private int Attack;

    public Character(string name, int life, int damage)
    {
        this.name = name;
        this.life = life;
        this.damage = damage;
        this.hasAnEffect = false;
    }

    public void PrintStats()
    {
        Console.WriteLine("Nombre: ", this.name);
        Console.WriteLine("PS: ", this.life);
        Console.WriteLine(" ");
    }

    public void SetAttack(int attack)
    {
        this.Attack = attack;
    }

    public void SetDamage(int attack)
    {
        this.life = this.life - attack;

        if (this.life < 0)
        {
            this.life = 0;
        }
    }
}

File: wodens_journey_game/Character/Character.py
class Character:
    
    def __init__(self, name, life, damage):
        
        self.name = name
        self.life = life
        self.damage = damage
        self.has_an_effect = False
        self.attack = 0
    
    def print_stats(self):
        
        print(f"Nombre: {self.name}")
        print(f"HP: {self.life}")
        print(" ")
    
    def set_attack(self, attack):
        
        self.attack = attack
    
    def get_damage(self, attack):
        
        self.life = self.life - attack

        if self.life < 0:
            
            self.life = 0